src/swap/SwapScreen.tsx

```diff
@@ -75,12 +59,19 @@ export function SwapScreenSection({ showDrawerTopNav }: { showDrawerTopNav: bool
     ExperimentConfigs[StatsigExperiments.SWAPPING_NON_NATIVE_TOKENS]
   )
 
-  const supportedTokens = useSelector(
-    swappingNonNativeTokensEnabled ? swappableTokensSelector : coreTokensSelector
-  )
+  // sorted by USD balance and then alphabetical
+  const supportedTokens = useSelector(swappableTokensSelector)
+  const swappableTokens = useMemo(() => {
+    const tokensWithUsdPrice = supportedTokens.filter(
+      (token) => token.isCoreToken && token.usdPrice && token.usdPrice.gt(0)
```

Comment:
I think `token.isCoreToken` shouldn't be here, right? We want all the tokens that have an usdPrice regardless of whether it's native or not.

Reply:
AH thanks, copy + paste error 😅